Client/Assets/CustomSearchHelper.js

```diff
@@ -0,0 +1,44 @@
+//xcode is soooo garbage with js. whyyyyyyyyy
```

Comment:
Yeah, it is :( I recommend using an external text editor if you aren't already.

But: please remove this comment and replace with the MPL license block header.